packages/aws-cdk/test/integ/run-wrappers/repo/npm

```diff
@@ -5,7 +5,11 @@ command=$1
 
 lerna=$REPO_ROOT/node_modules/.bin/lerna
 
-if [[ "$command" == "install" || "$command" == "i" ]]; then
+if [ ! -z ${USE_PUBLISHED_FRAMEWORK_VERSION:-} ]; then
```

Comment:
Feels convoluted. Why do we need the npm wrapper in this case at all?

Reply:
We don't really. This just seemed to be the most minimal change needed to get this working.
Let's revisit this at a later date? I feel like any solution won't be too great since this entire area needs some work.